server.py

```
@@ -0,0 +1,4 @@
+from __future__ import unicode_literals
+from ably import AblyRest
+
+client = AblyRest('{{ApiKey}}')
```

As with the other tutorial, we need to raise an exception if the API Key has not been set with instructions for the user to add the API key